lib/note-content-editor.tsx

```diff
@@ -172,9 +172,7 @@ class NoteContentEditor extends Component<Props> {
         });
       }
     }, SPEED_DELAY);
-    if (document?.activeElement?.id !== 'search-field') {
-      this.focusEditor();
-    }
+    this.focusEditor();
```

Comment:
Doesn't this go back to focusing the editor on load without any checks?

Reply:
It does, but I don't think this check is necessary. In most cases, I think we want the editor to have focus on load. We just don't want it to unexpectedly take focus if we've clicked the search field. 
I believe the focus was getting taken not from the initial on load, but the logic check to see if the selected search index had changed was always executing on first load because it was comparing `null` to `0` for the `prevProps.selectedSearchMatchIndex` and `props.selectedSearchMatchIndex`.

I've resolved that by setting the default to be `null`. That fixed another bug as well where the first time you pressed `CtrlOrCmd + g` to go through the search results it would select the second instance instead of the first.